Resources/views/Role/view_role.html.twig

```diff
@@ -21,7 +21,15 @@
     <section class="ez-tabs ez-serverside-content">
         <ul class="ez-tabs-list">
             <li class="ez-tabs-label is-tab-selected"><a href="#ez-tabs-role-name">{{ role.identifier }}</a></li>
-            <li class="ez-tabs-label"><a href="#ez-tabs-content">{{ 'assignment.user_or_group_using'|trans({'%role%': role.identifier, '%count%': role_assignments|length }) }}</a></li>
+            <li class="ez-tabs-label">
+                <a href="#ez-tabs-content"
+                   {%- if tab_id == 'ez-tabs-content' -%}
+                   id="ez-tabs-activate-on-load"
```

Comment:
@dpobel Indeed, that is easier :) Still struggling with the reload after assignment create, which is supposed to reload the page with a different `tabId` parameter. This doesn't seem to do anything:
`this.get('app').navigateTo('admin_roleView', {roleId: data.roleId, tabId: 'ez-tabs-content'});`